src/assets/translations/en/main.json

```diff
@@ -399,7 +399,7 @@
       "saversVaults": {
         "description": "Earn yield on your %{asset} through the THORChain Savers Vaults. There is no lock up time, and rewards are accumulated automatically. Yield on THORChain comes from swap fees and block rewards. Single-sided yield is capped at half the double-sided yield of liquidity providers.",
         "runePoolDescription": "RUNEPool lets you earn more %{asset} by pooling your %{asset} with others. You don't have to pick specific pools, everyone shares in the overall performance.",
-        "runePoolOverviewDescription": "RUNEPool optimizes RUNE usage by depositing it into all POL-enabled liquidity pools in THORChain. By pooling RUNE across these pools, participants earn the combined APY but face exposure to the aggregate impermanent loss. Instead of choosing individual pools, users benefit from the overall performance, effectively investing in a RUNE index and all THORChain PoL-enabled assets. This simplifies liquidity provision and reduces individual risk.",
+        "runePoolOverviewDescription": "RUNEPool optimizes %{asset} usage by depositing it into all POL-enabled liquidity pools in THORChain. By pooling RUNE across these pools, participants earn the combined APY but face exposure to the aggregate impermanent loss. Instead of choosing individual pools, users benefit from the overall performance, effectively investing in a %{asset} index and all THORChain PoL-enabled assets. This simplifies liquidity provision and reduces individual risk.",
```

Comment:
Think it's fine to leave as-is since RUNE is the `assetId` / `underlyingAssetId` of RUNEPool, but happy either way

Reply:
I don't really mind on my side, I did it to make you happy regarding the comment where we introduce this static change